### ISSUE TITLE
Literal List circuit component can't return a list of type table

### ISSUE THREAD

**Issue by FA120** [tgstation/tgstation#61983]
## Testmerges:
#61917 

## Reproduction:
Make a list literal component , set the type to 'table' and link a table to one of the index input port, the 'value' output port stays 'null' no matter how many time you trigger the component
![List Literal](https://user-images.githubusercontent.com/91917171/136627782-676669b1-4fe4-4009-837c-1891da3ab39b.PNG)

**Comment by FA120** [comment]
After some local testing local testing with admin powers I found out what exactly the problem is but since I am no coder,I can't help in fixing it but I thought explaining it would help:
Tables are,as you know,lists of lists,and apparently you could put a list into a list through list literal and that stuff,but if you try to put a list of list inside a list,the value becomes null...Even without using the table variable,I tried to make a list(X) using a literal list,then put it inside another list(Y) through literal list,it worked fine...But when I tried to put the new list(Y that contains X) in a new list Z(so Z[1]=Y) Z becomes null

**Comment by Watermelon914** [comment]
yeah.. it shouldn't be able to select table in the first place. Damn
